shell/platform/android/test_runner/build.gradle

```diff
@@ -4,7 +4,7 @@ buildscript {
     mavenCentral()
   }
   dependencies {
-    classpath "com.android.tools.build:gradle:7.0.0"
+    classpath "com.android.tools.build:gradle:7.3.0"
```

Comment:
should this be 8.3?

Reply:
No, this is the AGP version not the gradle version